x/ibc/03-connection/client/utils/utils.go

```diff
@@ -82,3 +87,37 @@ func QueryClientConnections(
 	connPathsRes := types.NewClientConnectionsResponse(clientID, paths, res.Proof, res.Height)
 	return connPathsRes, nil
 }
+
+// ParsePrefix unmarshals an cmd input argument from a JSON string to a commitment
+// Prefix. If the input is not a JSON, it looks for a path to the JSON file.
+func ParsePrefix(cdc *codec.Codec, arg string) (commitment.Prefix, error) {
```

Comment:
This helper function could be generalized imo, but I'm not sure if it's the right approach as the cmd description states that it receives a path to the prefix as an argument while in reality, we unmarshal the input and then we check the existence of a path to the JSON file. 

@AdityaSripal thoughts?